lib/cn/csched_sp3.c

```diff
@@ -1587,23 +1602,25 @@ sp3_check_roots(struct sp3 *sp)
     list_for_each_entry (tree, &sp->mon_tlist, ct_sched.sp3t.spt_tlink) {
 
         bool have_work;
-        uint qnum;
 
         lvl_max = max(lvl_max, tree->ct_lvl_max);
 
-        if (sp3_work(tn2spn(tree->ct_root), &sp->thresh, wtype_rspill, debug, &qnum, &sp->wp))
+        if (sp3_work(tn2spn(tree->ct_root), &sp->thresh, wtype_rspill, debug, &sp->wp))
```

Comment:
sp3_work() no longer picks the scheduling queue, that is now done by sp3_schedule().

Reply:
Good. I think?  There was some duplicate logic here right?

Reply:
Yeah, the job and queue logic is now completely managed by sp3_schedule(), sp3_work() no longer has any visibility or say in the matter...